### PR TITLE
Add missing link_name for getmntinfo

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Amanieu

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Fixed format.

**Comment by GuillaumeGomez** [comment]
@asomers Do you know what's going on in here by any chance?

File: src/unix/bsd/freebsdlike/freebsd/mod.rs [hunk 1]
```diff
@@ -4037,6 +4037,10 @@ extern "C" {
     pub fn getfh(path: *const ::c_char, fhp: *mut fhandle_t) -> ::c_int;
     pub fn lgetfh(path: *const ::c_char, fhp: *mut fhandle_t) -> ::c_int;
     pub fn getfsstat(buf: *mut ::statfs, bufsize: ::c_long, mode: ::c_int) -> ::c_int;
+    #[cfg_attr(
+        all(target_os = "freebsd", freebsd11),
+        link_name = "getmnntinfo@FBSD_1.0"
```
Comment (asomers) [inline src/unix/bsd/freebsdlike/freebsd/mod.rs:4042]:
```suggestion
        link_name = "getmntinfo@FBSD_1.0"
```

Comment (GuillaumeGomez) [reply src/unix/bsd/freebsdlike/freebsd/mod.rs]: I feel very stupid right now... Thanks!

**Pull request merged**